LICENSE

```diff
@@ -527,3 +527,46 @@ For
    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
    TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
    THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+For
+  xz-<version>.jar:
+
+    http://git.tukaani.org/?p=xz-java.git;a=blob_plain;f=COPYING
```

Comment:
No need to specify the URL here; it may even change in the future since this doesn't point to a git hash or anything